waspc/packages/deploy/src/providers/fly/deploy/DeployOptions.d.ts

```diff
@@ -0,0 +1,7 @@
+import { CommonOptions, LocalBuildOptions } from '../../shared/CommonOptions';
+
+export interface FlyDeployOptions extends CommonOptions, LocalBuildOptions {
```

Comment:
I like the renaming of types to be provider-specific, to help distinguish/searching. Is there any way we may be able to encode this convention into the type system? Just thinking out loud to make sure all providers "do the right thing" here.

Reply:
Yeah, agreed, let's keep it simple for now and rely on conventions. 👍🏻